i18next-scanner.config.js

```diff
@@ -0,0 +1,71 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+const fs = require('fs');
+const chalk = require('chalk');
+
+// list of func used to 
+// mark the strings for translation
+const funcList = ['i18next.t', 'i18n.t', 't'];
+
+// available languages
+const languages = ['en','de', 'tr'];
```

Comment:
That can work for now but we should think what are the needed steps to add a new language to this repository. Maybe we could create an issue for now explaining the process...